src/components/Table/Filters.tsx

```diff
@@ -146,7 +148,7 @@ const FilterWrapper = styled.div`
 `;
 
 interface StyledOutlinedProps {
-  active?: boolean;
+  active?: number;
```

Comment:
These error messages were clogging up my console so I fixed this file, but can change it back if you want me to 

<img width="472" alt="Screen Shot 2022-02-09 at 9 50 36 AM" src="https://user-images.githubusercontent.com/47064971/153433705-5876411c-0e1a-4690-982e-c11efc9d5bd3.png">